packages/acpc/acpc.1.1.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/UnixJunkie/ACPC"
+dev-repo: "git://github.com/UnixJunkie/ACPC"
+bug-reports: "https://github.com/UnixJunkie/ACPC/issues"
+license: "BSD"
+build: [
+  ["obuild" "configure"]
+  ["obuild" "build"]
+]
+install: [
+  ["cp" "dist/build/acpc/acpc"
```

Comment:
This might be better replaced by a `acpc.install` file (that would be located in `acpc/acpc.1.1.1/files`)